frame/nfts/src/lib.rs

```diff
@@ -61,6 +61,26 @@ pub use weights::WeightInfo;
 
 type AccountIdLookupOf<T> = <<T as frame_system::Config>::Lookup as StaticLookup>::Source;
 
+pub trait Incrementable {
+	fn increment(&self) -> Self;
+}
+
+macro_rules! impl_incrementable {
+	($type:ty) => {
```

Comment:
```suggestion
	(($type:ty),+) => {
```

So that you can simply write `impl_incrementable!(u8, u16, ...)`.